### PR TITLE
ACS-3363 Output TAS logs for failed tests.

### PR DESCRIPTION
**Pull request by tpage-alfresco**

### CODE REVIEW

**Comment by tpage-alfresco** [comment]
For an example of this being used see line 1950 onwards of this job: https://app.travis-ci.com/github/Alfresco/alfresco-community-repo/jobs/582941560

**Review by mpichura — APPROVED** [review]
Good idea

**Review by KikoUK — APPROVED** [review]

File: packaging/tests/scripts/output_logs_for_failures.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+TAS_DIRECTORY=$1
+
+cd ${TAS_DIRECTORY}
+
+failures=$(grep 'status="FAIL"' target/surefire-reports/testng-results.xml | sed 's|^.*[ ]name="\([^"]*\)".*$|\1|g')
```
Comment (KikoUK) [inline packaging/tests/scripts/output_logs_for_failures.sh:7]: What does the `sed `part of this line do?

Comment (tpage-alfresco) [reply packaging/tests/scripts/output_logs_for_failures.sh:7]: The testng-results.xml file contains lines that look like:
```
<test-method signature="getRuleSetAndInheritedBy()[pri:0, instance:org.alfresco.rest.rules.GetRuleSetsTests@613a8ee1]" started-at="2022-09-16T09:48:12Z" name="getRuleSetAndInheritedBy" test-instance-name="" finished-at="2022-09-16T09:48:12Z" duration-ms="774" status="FAIL">
```

The sed command retrieves the `name="getRuleSetAndInheritedBy"` section and extracts the method name.

This is then matched in `TestSuite-output.txt` which contains lines like:
```
[main] 09:48:12 INFO  RestTest:77 - *** STARTING Test: [getRuleSetAndInheritedBy] ***
```

**Pull request merged**